Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -27,19 +27,31 @@ extension OfflineMapAreasView {
         /// The offline map task.
         private let offlineMapTask: OfflineMapTask
         
-        /// The preplanned offline map information.
+        /// The preplanned map information.
         @Published private(set) var preplannedMapModels: Result<[PreplannedMapModel], Error>?
         
+        /// The offline preplanned map information sourced from downloaded mobile map packages.
+        @Published private(set) var offlinePreplannedMapModels: [PreplannedMapModel]?
```

Comment:
I'm a bit wary whether this needs to be optional or not. The online `preplannedMapModels` uses `nil` to convey that it is still loading. Whereas the loading state is not reflected in the UI for offline models.

What about adding a progress view for the offline models as well, even if it might never be visible thanks to very fast loading from disk?

Otherwise, we can use its emptiness to convey whether there are local areas or not.

Reply:
I added a progress view which is presented when the offlinePreplannedMapModels are `nil`. The progress view is not visible since the files load from disk quickly, but is good to have in case the files do not load quickly.